src/Storage/Listener/LocalListener.php

```diff
@@ -29,7 +27,7 @@ class LocalListener extends AbstractListener {
  * @var array
  */
 	protected $_defaultConfig = [
-		'pathBuilder' => 'BasePath',
+		'pathBuilder' => 'Local',
```

Comment:
Do as you wish then, you could try it with the property first, the MergeVars trait should make it possible to avoid overriding the default properties. Or add an initialize() method for the path builder as well. I'm not a big fan of overriding constructors, having a separate method to inject custom "start up" logic is more clean IMHO.